app/ui/mainwindow2.ui

```diff
@@ -81,6 +82,7 @@
      <addaction name="actionExport_Animated_GIF"/>
      <addaction name="separator"/>
      <addaction name="actionExport_Palette"/>
+     <addaction name="actionExport_Other_Palette_formats"/>
```

Comment:
I don't see a need to have multiple export actions, just provide one "Export Palette" then let the dialog. Remove the Export palette and keep the dialog where you can select extension.